config/flags_payments.go

```diff
@@ -65,6 +65,18 @@ var (
 		Name:  "payments.disable",
```

Comment:
Do we need to add this empty file `config/config.toml`?

Reply:
No we don't, thx :)